mycroft/configuration/config.py

```diff
@@ -249,6 +249,9 @@ def load_config_stack(configs=None, cache=False, remote=True):
         xdg_locations = get_xdg_config_locations()
 
         if not configs:
+            LOG.debug("Loading configs")
+            LOG.debug(f"DEFAULT_CONFIG={DEFAULT_CONFIG}")
+            LOG.debug(f"SYSTEM_CONFIG={SYSTEM_CONFIG}")
```

Comment:
please remove these, we have enough log pollution around config loading :P

Reply:
whoops, accidental testing leftover from weeks ago..